src/Generators/AbstractTagGenerator.php

```diff
@@ -176,21 +178,32 @@ public function getExistingTags()
     /**
      * Generate the Owner-properties for extensions.
      *
-     * @throws NotFoundExceptionInterface
+     * @throws ReflectionException
      */
     protected function generateOwnerTags()
     {
         $className = $this->className;
         // If className is abstract, Injector will fail to instantiate it
-        $reflection = new \ReflectionClass($className);
+        $reflection = new ReflectionClass($className);
         if ($reflection->isAbstract()) {
             return;
         }
         if (Injector::inst()->get($this->className) instanceof Extension) {
             $owners = array_filter(DataObjectAnnotator::getExtensionClasses(), function ($class) use ($className) {
-                $config = Config::inst()->get($class, 'extensions');
-
-                return ($config !== null && in_array($className, $config, null));
+                $config = Config::inst()->get(
+                    $class,
+                    'extensions',
+                    Config::UNINHERITED | Config::EXCLUDE_EXTRA_SOURCES
+                );
+                if (!$config) {
+                    return false;
```

Comment:
Return false?? The tests show it's correct, but it looks a bit weird to me.

Reply:
That's a class with an empty extensions config :) E.g. `private static $extensions = [];`